src/Options/Definitions.js

```diff
@@ -148,16 +148,16 @@ module.exports.ParseServerOptions = {
   userSensitiveFields: {
     env: 'PARSE_SERVER_USER_SENSITIVE_FIELDS',
     help:
-      'Personally identifiable information fields in the user table the should be removed for non-authorized users. **Deprecated** @see protectedFields',
+      'Personally identifiable information fields in the user table the should be removed for non-authorized users. Deprecated @see protectedFields',
     action: parsers.arrayParser,
     default: ['email'],
   },
   protectedFields: {
     env: 'PARSE_SERVER_PROTECTED_FIELDS',
     help:
-      'Personally identifiable information fields in the user table the should be removed for non-authorized users.',
+      'Protected fields that should be treated with extra security when fetching details.',
     action: parsers.objectParser,
-    default: { _User: { '*': ['email'] } },
+    default: [],
```

Comment:
why the change to the default?

Reply:
This is autogenerated. ObjectParser doesn't exist for this json. I can fix this in a separate PR.